packages/commerce-server/src/record-new-purchase.ts

```diff
@@ -142,6 +143,11 @@ export async function recordNewPurchase(checkoutSessionId: string): Promise<{
     purchaseType = NEW_INDIVIDUAL_PURCHASE
   }
 
+  await createPurchaseUserTransfer({
+    purchaseId: purchase.id,
+    sourceUserId: user.id,
+  })
```

Comment:
You could push this up into `createMerchantChargeAndPurchase` in `prisma-api.ts`. Then it is closer to the actual purchase creation and you can bundle it into the database transaction that the other purchase concerns are wrapped in:

<img width="703" alt="CleanShot 2023-02-01 at 13 45 15@2x" src="https://user-images.githubusercontent.com/694063/216147187-c6b5600f-1fd5-4522-8ca1-f20dd10acaf2.png">